### PR TITLE
feat: Review UX to add node - MEED-6834 - Meeds-io/MIPs#137

### PR DESCRIPTION
**Pull request by SaraBoutej**

### CODE REVIEW

File: layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default {
       return this.$t('siteNavigation.addElementDrawer.title');
     },
     disabled() {
-      return !this.isValidForm || !this.pageTemplate || false;
+      return !this.isValidForm || this.pageTemplate === 'empty'  || false;
```
Comment (boubaker) [inline layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue]: may be delete the default value 'empty' instead, WDYT ?

File: layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue [hunk 2]
```diff
@@ -86,7 +86,7 @@ export default {
       return this.$t('siteNavigation.addElementDrawer.title');
     },
     disabled() {
-      return !this.isValidForm || !this.pageTemplate || false;
+      return !this.isValidForm || !this.pageTemplate  || false;
```
Comment (boubaker) [inline layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue]: Extra space, please run `npm run lint` as well to automatically fix another spacing problem.
 
```suggestion
      return !this.isValidForm || !this.pageTemplate || false;
```

Comment (SaraBoutej) [reply layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue]: done

File: layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue (hunk 1 above)
Comment (SaraBoutej) [reply layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue]: done

**Review by boubaker — DISMISSED** [review]

File: layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationExistingPageElement.vue [hunk 3]
```diff
@@ -31,13 +31,13 @@
...
       :all-sites="allSites"
-      class="mb-6" />
+      :class="!allSites && 'mb-8' || ''" />
     <site-navigation-page-suggester
       :page="selectedPage"
       :all-sites="allSites"
       :site-type="selectedSiteNavigation?.siteType"
       :site-name="selectedSiteNavigation?.name"
-      class="mb-8" />
+      class="" />
```
Comment (boubaker) [inline layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationExistingPageElement.vue]: Useless
```suggestion
      />
```

File: layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationExistingPageElement.vue [hunk 4]
```diff
@@ -31,13 +31,13 @@
       v-if="!allSites"
       v-model="selectedSiteNavigation"
       :all-sites="allSites"
-      class="mb-6" />
+      :class="!allSites && 'mb-8' || ''" />
```
Comment (boubaker) [inline layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationExistingPageElement.vue]: ` || ''` not needed

```suggestion
      :class="!allSites && 'mb-8'" />
```

File: layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationExistingPageElement.vue (hunk 3 above)
Comment (SaraBoutej) [reply layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationExistingPageElement.vue]: done

File: layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationExistingPageElement.vue (hunk 4 above)
Comment (SaraBoutej) [reply layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationExistingPageElement.vue]: done

**Review by boubaker — APPROVED** [review]

**Pull request merged**